src/t_list.c

```diff
@@ -95,58 +94,70 @@ void listTypeTryConversionForValues(robj *o, robj **argv, int start, int end,
 }
 
 /* Check the length and size of a listpack to see if we need to convert it to quicklist. */
```

Comment:
comment should now say "from listpack", not "to quicklist", right?
same for the other function below..

Reply:
Fixed in [`4ac67aa` (#11303)](https://github.com/redis/redis/pull/11303/commits/4ac67aa0866e9ab9a1c4c588f5da9a2e660972c4)